receiver/azureeventhubreceiver/eventhubhandler_test.go

```diff
@@ -57,15 +57,20 @@ func (m mockListenerHandleWrapper) Err() error {
 }
 
 type mockDataConsumer struct {
-	logsUnmarshaler  eventLogsUnmarshaler
-	nextLogsConsumer consumer.Logs
-	obsrecv          *receiverhelper.ObsReport
+	logsUnmarshaler   eventLogsUnmarshaler
+	nextLogsConsumer  consumer.Logs
+	nexTracesConsumer consumer.Traces
```

Comment:
There's a minor typo here. It should be nextTracesConsumer.

Reply:
typo fixed!